test/integration_test.sh

```diff
@@ -30,6 +30,13 @@ trap finish SIGINT SIGTERM INT TERM EXIT
 # -------------------------------------------------------------------------------------------------
 
 
+
+if ! [ -x "$(command -v docker-compose)" ]; then
```

Comment:
this might fail if docker-compose is an alias.
Unlikely, but how about:
```suggestion
if ! command -v docker-compose; then
```

Reply:
Makes sense 👍